daemons/controld/controld_fencing.c

```diff
@@ -618,6 +618,16 @@ static gboolean
 te_connect_stonith(gpointer user_data)
 {
     int rc = pcmk_ok;
+    static time_t last_attempt = 0;
+    time_t now = time(NULL);
+
+    /* This callback can get called a LOT if, eg, pacemaker_fenced
+     * goes down, so be a bit more restrained trying to reconnect
+     */
+    if (user_data && now <= last_attempt) {
+        return TRUE;
```

Comment:
hmm wouldn't that stop further retries?
we could arm the trigger anew or return -1 but that would still keep the trigger spinning - right.
so we would probably rather have to arm a timer.

Reply:
maybe we could set this as a timer rather than a trigger, and use a very short time on the initial creation